articles/user-guide/libraries/machine-learning/basic-classification.md

```diff
@@ -13,80 +13,90 @@ uid: microsoft.quantum.libraries.overview.machine-learning.basics
+In this Quickstart, you will learn how to run a quantum sequential classifier written in Q# using the Quantum Machine Learning library of the QDK.
 
-In this guide we will use the half-moon dataset, using a classifier structure defined in Q#.
+In this guide we will train a simple sequential model using a classifier structure defined in Q#. The model is trained on a half-moon dataset with training and validation data that you can find in the [code samples](https://github.com/microsoft/Quantum/tree/main/samples/machine-learning/half-moons). We will create our Q# project using either a Pyhton or a C# to load data and that call Q# operations. 
 
 ## Prerequisites
 
 - The Microsoft [Quantum Development Kit](xref:microsoft.quantum.install-qdk.overview).
 - Create a Q# project for either a [Python host program](xref:microsoft.quantum.install-qdk.overview.python) or a [C# host program](xref:microsoft.quantum.install-qdk.overview.cs).
+- To add the [Microsoft.Quantum.MachineLearning](https://www.nuget.org/packages/Microsoft.Quantum.MachineLearning) package to your Q# project, run the following command:
+
+```dotnetcli
+dotnet add package Microsoft.Quantum.MachineLearning
+```
+
+## Q\# classifier code
+
+We start by creating a a file called `Training.qs`and adding the following code to it:
+
+:::code language="qsharp" source="~/quantum/samples/machine-learning/half-moons/Training.qs" range="4-":::
+
+The most important functions and operations defined in the code above are:
+
+- `ClassifierStructure() : ControlledRotation[]` : in this function we set the structure of our circuit model by adding the layers of the controlled gates we consider. This step is analogous to the declaration of layers of neurons in a sequential deep learning model.
+- `TrainHalfMoonModel() : (Double[], Double)` : this operation is the core part of the code and defines the training. Here we load the samples from the dataset included in the library, we set the hyper parameters and the initial parameters for the training and we start the training by calling the operation `TrainSequentialClassifier` included in the library. It outputs the parameters and the bias that determine the classifier.
+- `ValidateHalfMoonModel(parameters : Double[], bias : Double) : Int` : this operation defines the validation process to evaluate the model. Here we load the samples for validation, the number of measurements per sample and the tolerance. It outputs the number of misclassifications on the chosen batch of samples for validation.
 
 ## Host program
 
-Your host program consists of three parts:
+Next, in the same folder we create a host program. Your host program consists of three parts:
 
-- Load the dataset and choose a set of starting parameters for your model.
+- Load the dataset `data.json` and choose a set of classifier parameters where you want to start your training iterations for your model.
```

Comment:
I think the last addition to this sentence makes it more complicated without adding new information compared to the previous formulation.

Reply:
With this sentence I tried to explain why the host file starts by choosing some parameters of the dataset, instead of taking the whole set.